src/main.js

```diff
@@ -88,39 +77,26 @@ Vue.use(VSanitize, {
 // 注册全局事件总线
 Vue.prototype.$eventBus = eventBus
 
-new Vue({
-  el: '#app',
-  i18n,
-  router,
-  store,
-  mounted() {
-    // 移除加载页面
-    const loadingElement = document.getElementById('loading')
-    if (loadingElement) {
-      setTimeout(() => {
-        loadingElement.style.display = 'none'
-      }, 500)
-    }
-  },
-  render: h => h(App)
-})
-
-;(function() {
-  request({
-    url: '/api/v1/authentication/user-session/',
-    method: 'get',
-    disableFlashErrorMsg: true
+async function initApp() {
+  await fetchTranslationsFromAPI()
+  watchSessions()
+
+  new Vue({
+    el: '#app',
+    i18n,
+    router,
+    store,
+    mounted() {
+      // 移除加载页面
+      const loadingElement = document.getElementById('loading')
+      if (loadingElement) {
+        setTimeout(() => {
+          loadingElement.style.display = 'none'
+        }, 500)
+      }
+    },
+    render: h => h(App)
   })
-})()
-
-let IdBeforeunload = false
+}
 
-window.addEventListener('beforeunload', (event) => {
-  if (IdBeforeunload) return
-  IdBeforeunload = true
-  request({
-    url: '/api/v1/authentication/user-session/',
-    method: 'delete',
-    disableFlashErrorMsg: true
-  })
-})
+initApp().then()
```

Comment:
It does not contain any errors or issues based on my knowledge cutoff until September 1, 2021. However, you should always verify new changes with latest documentation updates or official bug fixes from developers like myself to ensure the compatibility and security of applications.